src/App.tsx

```diff
@@ -1,13 +1,33 @@
-import React from 'react';
+import React, {useState} from 'react';
 import classes from "./App.module.scss";
 import Header from "./components/Header";
 import StartPage from "./components/pages/StartPage";
+import {IOptions} from "./interfaces/options";
+import TextPage from "./components/pages/TextPage";
+import FinishPage from "./components/pages/FinishPage";
 
 const App = () => {
+    const [options, setOptions] = useState<IOptions>({
```

Comment:
Here initial state is unnecessary and excessive because it is not used anywhere and is just overwritten when you press the start button . All controls already have its initial states.